commons/all/src/main/java/org/infinispan/commons/util/Version.java

```diff
@@ -161,7 +168,7 @@ public static void printFullVersionInformation() {
     * Returns version information as a string.
     */
    public static String printVersion() {
-      return INSTANCE.brandname + " '" + INSTANCE.codename + "' " + INSTANCE.version;
+      return INSTANCE.brandName + " '" + INSTANCE.codename + "' " + INSTANCE.version;
```

Comment:
I think it should be `Infinispan` instead of the brand name. 
```suggestion
      return "Infinispan '" + INSTANCE.codename + "' " + INSTANCE.version;
```

This is used in CLI (`version` command) and it is logged when the server starts:
`ISPN000128: Infinispan version: My Brand 'N/A' 11.0.0-SNAPSHOT`

Reply:
or change its format:
`<brand> <brand_version> (Infinispan <project_version>)`

Reply:
May I suggest to move the codename to the end and use the brand version instead:
```suggestion
      return INSTANCE.brandName + " " + INSTANCE.brandVersion + " (" + INSTANCE.codename + ")";
```